runtime/sema/errors.go

```diff
@@ -53,6 +53,27 @@ func (e *unsupportedOperation) Error() string {
 	)
 }
 
+// InvalidPragmaError
+
+type InvalidPragmaError struct {
+	Message string
+	ast.Range
+}
+
+func (e *InvalidPragmaError) isSemanticError() {}
+
+func (e *InvalidPragmaError) StartPosition() ast.Position {
+	return e.Range.StartPos
+}
+
+func (e *InvalidPragmaError) EndPosition() ast.Position {
+	return e.Range.EndPos
+}
```

Comment:
These are not needed, embedding `ast.Range` makes `InvalidPragmaError` implement the functions already 🎉 

```suggestion
```